proposals/unsafe-evolution.md

```diff
@@ -0,0 +1,138 @@
+# Unsafe Evolution
+
+Champion issue: https://github.com/dotnet/csharplang/issues/9704
+
+## Summary
+
+We update the definition of `unsafe` in C# from refering to locations where pointer types are used, to be locations where memory unmanaged by the runtime is dereferenced. These locations
+are where memory unsafety occurs, and are responsible for the bulk of CVEs categorized as memory safety issues.
```

Comment:
> These locations are where memory unsafety occurs

Could be nice to explain why. It's generally because one could be using free'd memory, right?

Reply:
That's motivation, not summary.

Reply:
Explaining this in motivation works for me. I would just like to see some "framework" of what is considered unsafe (use after free, double free, ...), somewhere near the top of the proposal, which the rest of the proposal is based on (and can be reviewed against).